core/src/mindustry/world/Block.java

```diff
@@ -123,7 +123,7 @@ public class Block extends UnlockableContent implements Senseable{
     /** for static blocks only: if true, tile data() is saved in world data. */
     public boolean saveData;
     /** whether you can break this with rightclick */
-    public boolean breakable;
+    public boolean breakable = true;
```

Comment:
This should be false for most blocks, and is a massive breaking change. I won't be accepting core logic changes like this.